bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -87,10 +89,14 @@ class AudioManager {
     this.isWaitingPermissions = false;
     this.devicesInitialized = false;
 
-    return Promise.all([
-      this.setDefaultInputDevice(),
-      this.setDefaultOutputDevice(),
-    ]).then(() => {
+    // Avoid ask microphone permission for "Listen Only"
+    const devicesInitializePromises = [];
+    if (this.isListenOnly == false) devicesInitializePromises.push(this.setDefaultInputDevice());
```

Comment:
Why did you choose to create this `if` case inside the `askDevicesPermission` method to avoid requesting the device on webkit implementations? You could just remove the forced permission request on line 155.

Reply:
I kept the `if` in this part to be the least intrusive possible in this change.
The function `askDevicesPermissions` is still calling the `setDefaultOutputDevice`, and setting other variables that I didn't know if is necessary.

Did you make tests without the `askDevicesPermissions` being called? so I can change the `if` location.